SPECS/nmi/nmi.spec

```diff
@@ -0,0 +1,54 @@
+%global debug_package %{nil}
+
+Summary:        Node Managed Identity
+Name:           nmi
+Version:        1.7.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Libraries
+URL:            https://github.com/Azure/aad-pod-identity
+#Source0:       https://github.com/Azure/aad-pod-identity/archive/refs/tags/v%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+Source1:        %{name}-%{version}-vendor.tar.gz
```

Comment:
Please provide instructions on how to generate the vendor tarball in a **reproducible** (see: `protobuf.spec` or `coredns-1.7.0.spec` from `1.0-dev` for instance) way, so that we can later recreate it and know how to generate a new one when we update the version.

In general we should never use a source file we don't know how to recreate in a consistent way or how to update it if needed.